packages/swapper/src/swappers/JupiterSwapper/errors.ts

```diff
@@ -0,0 +1,23 @@
+import type { SwapperErrorMapping } from '../baseErrors'
+import { SWAPPER_ERRORS } from '../baseErrors'
+
+export const JUPITER_ERROR_VALUES = {
```

Comment:
Not sure about the naming here, this is rather a subtring check , maybe something like `pattern` vernacular would be a better fit?

Reply:
This also looks better suited to an enum.

Reply:
Changed in [17f168a](https://github.com/shapeshift/web/pull/8433/commits/17f168a1ba52efd3e9b3f5e55e1b8f1e2266db37)